live-examples/js-examples/map/map.js

```diff
@@ -0,0 +1,17 @@
+const MAP = new Map();
+
+MAP.set('a', 1);
+MAP.set('b', 2);
+MAP.set('c', 3);
+
+let result = MAP.get('a');
+
+console.log(result);
+// expected output: 1
+
+MAP.set('a', 97);
+
+result = MAP.get('a');
+
+console.log(result);
```

Comment:
I think in these cases it would be fine to save some space by omitting the `result` variable:

```js
 console.log(MAP.get('a'));
```

Reply:
Okay. b2239e6 should address this.